src/SprykerEco/Yves/Payone/Form/DataProvider/BancontactOnlineTransferDataProvider.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Payone\Form\DataProvider;
+
+use Generated\Shared\Transfer\PaymentTransfer;
+use Generated\Shared\Transfer\PayonePaymentTransfer;
+use Spryker\Shared\Kernel\Transfer\AbstractTransfer;
+use Spryker\Yves\StepEngine\Dependency\Form\StepEngineFormDataProviderInterface;
+use SprykerEco\Shared\Payone\PayoneConfig;
+use SprykerEco\Yves\Payone\Form\BancontactOnlineTransferSubForm;
+
+class BancontactOnlineTransferDataProvider implements StepEngineFormDataProviderInterface
+{
+    /**
+     * @param \Spryker\Shared\Kernel\Transfer\AbstractTransfer|\Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Spryker\Shared\Kernel\Transfer\AbstractTransfer
+     */
+    public function getData(AbstractTransfer $quoteTransfer): AbstractTransfer
+    {
+        if ($quoteTransfer->getPayment() === null) {
```

Comment:
invert condition please

Reply:
@PhilinTv what's your opinion?

Reply:
Agree, we apply "early return" pattern here.